config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-postsubmits.yaml

```diff
@@ -35,6 +35,42 @@ periodics:
     testgrid-tab-name: Kubernetes Master Driver Release 1.3
     testgrid-alert-email: gke-managed-storage-alerts@google.com
     description: Kubernetes Integration tests for Kubernetes Master branch and Driver Release 1.3
+- name: ci-gcp-compute-persistent-disk-csi-driver-release-1-4-k8s-master-integration
+  interval: 4h
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220214-0aa8fe1d3a-master
+      args:
+      - "--repo=sigs.k8s.io/gcp-compute-persistent-disk-csi-driver=release-1-4"
+      - "--root=/go/src"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--clean"
+      - "--timeout=180" # Minutes (120m runs consistently time out)
+      - "--scenario=execute"
+      - "--" # end bootstrap args, scenario args below
+      - "test/run-k8s-integration-ci.sh"
+      env:
+      - name: GCE_PD_OVERLAY_NAME
+        value: "stable-master"
```

Comment:
I can use the stable-master overlay instead of the `prow-gke-release-staging-rc-master` overlay since this is the same as `stable-master`. Both use latest release and stable sidecar versions.